elements/pl-sketch/pl-sketch.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable */
+
+var size = 5;
+var color = '#000000';
+var curTool = null;
+
+function changeTool(tool, sketchpad) {
+    if (tool === curTool) {
+        return;
+    }
+    if (tool === 'pen') {
+        size = 5;
+        curTool = 'pen';
+        $('#pen').css('background', '#C0C0C0');
+        $('#eraser').css('background', '#FAFAFA');
+        sketchpad.penSize = size;
+        sketchpad.color = color;
+    } else {
+        size = 30;
+        curTool = 'eraser';
+        $('#eraser').css('background', '#C0C0C0');
+        $('#pen').css('background', '#FAFAFA');
+        sketchpad.penSize = size;
+        sketchpad.color = '#FFF';
+    }
+}
+
+function updateColor(col, sketchpad) {
+    color = col;
+    $('.pl-sketch-color-icon').css('backgroundColor', color);
+    if (curTool === 'pen') {
+        sketchpad.color = color;
+    }
+}
+
+$(function() {
+    var width = 600;
+    var height = 700;
+    var sketches_string = $('#pl-sketch-json').html();
+    var sketchpad;
```

Comment:
We typically try to use `const` and `let` in new JavaScript code.  Also, I think the `width` and `height` would be good values to make attributes so that question authors can change them.